### DISCUSSION TITLE
How to ask for community help

### DISCUSSION THREAD

**Discussion by Vonng** [pgsty/pigsty#338] [Announcements]
When having troubles with pigsty. You can ask the community for help, with enough information context, here's a template 

**What happend?**  (**REQUIRED**)

```
```

**Pigsty Version & OS Version** (**REQUIRED**)

```
$ grep version  pigsty.yml 

$ cat /etc/os-release
```

**Pigsty Config File** (**REQUIRED**)

Don't forget to remove sensitive information like passwords, etc...

**How to reproduce it?**

```
```

**What did you expect to happend?**

```
```

**Monitoring Screenshots**

If you are using pigsty monitoring system, you can paste **RELEVANT** screenshots here.

```
```

**Error Log**

Please copy and paste any **RELEVANT** log output. Do not paste something like "Failed to start xxx service"

- Syslog:    `/var/log/messages` (rhel) or `/var/log/syslog` (debian)
- Postgres:   `/pg/log/postgres/*`
- Patroni:   `/pg/log/patroni/*`
- Pgbouncer:   `/pg/log/pgbouncer/*`
- Pgbackrest:   `/pg/log/pgbackrest/*`

```
journalctl -u patroni
journalctl -u <service name>
```

**Have you tried to search the issue?**

- [Github Issues](https://github.com/Vonng/pigsty/issues)
- [Pigsty Doc: FAQ](https://doc.pigsty.cc/#/FAQ)
- [Pigsty Discussion Forum](https://github.com/Vonng/pigsty/discussions)

- [ ] Yes

**Anything else we need to know?**

```
```

**Comment by Vonng** [comment]
中文版本：



当您使用 Pigsty 遇到问题时，可以向社区求助，在本论坛中，Issue里，或讨论群组中发问。请尽量提供足够的信息，以便社区成员帮助您解决问题。



**发生了什么事？**  (**必选项**)

```
```


**Pigsty版本号与操作系统版本号** (**必选项**)

```
$ grep version  pigsty.yml 

$ cat /etc/os-release
```


**Pigsty配置文件**

> 请不要忘记抹掉任何敏感信息：密码，内部密钥，敏感配置等。

```
```



**如何复现此问题？**


```
```



**你期待发生什么？**

```
```


**监控截图**

> 如果你在使用 Pigsty 提供的监控系统，可以提供 **相关** 的截图。


```
```


**错误日志**

请尽可能提供与错误有关的日志。请不要粘贴类似“Failed to start xxx service”之类没有帮助的内容。

您可以从 Grafana / Loki 中查询日志，或从以下位置获取日志：

- Syslog:    `/var/log/messages` (rhel) or `/var/log/syslog` (debian)
- Postgres:   `/pg/log/postgres/*`
- Patroni:   `/pg/log/patroni/*`
- Pgbouncer:   `/pg/log/pgbouncer/*`
- Pgbackrest:   `/pg/log/pgbackrest/*`

```
journalctl -u patroni
journalctl -u <service name>
```




**您已经搜索过 Issue/FAQ/论坛了吗？**

> 在 FAQ 中，我们提供了许多常见问题的解答，请在提问前检查 [Pigsty常见问题](https://doc.pigsty.cc/#/zh/FAQ)

您也可以从 Github Issue 与 Discussion 中搜索相关问题：

- [Github Issues](https://github.com/Vonng/pigsty/issues)
- [Pigsty Discussion Forum](https://github.com/Vonng/pigsty/discussions)



**有什么其他信息是我们需要知道的吗？**


```
```

**Comment by zhangguanzhang** [comment]
```
$ cat pigsty.yml  | grep version
```
change to
```
$ grep version  pigsty.yml 
```
👀
